packages/docs/docs/docs/new_2.3.md

```diff
@@ -0,0 +1,65 @@
+---
+id: new_2.3
+title: What's new in ZeppelinOS 2.3
+---
+
+Here you will find all the major new features shipped in version 2.3. Check out the changelog for a more detailed list of what has changed since 2.2.
+
+## Interactive commands
+
+Most `zos` CLI commands will now prompt for any information they need to work, and guide you through the whole process. This is a major improvement over version 2.2, where you needed to specify all arguments and options along with the command, or get an error back.
+
+Commands are also much better at fulfilling steps for you. In version 2.2, you would need to follow these steps to add a contract to your project, push it to a network, and then create and initialize a new instance:
+
+```
+$ zos add MyContract
+$ zos push --network mainnet
+$ zos create MyContract --network mainnet --init initialize --args 42
+> Instance created at 0x123456
+```
+
+Starting on version 2.3, you can just run `zos create` instead, and the CLI will guide you through the entire contract creation and initialization process.
+
+All previous arguments and flags still work. However, if you are writing a script to run in an automated environment, you can also add a `--no-interactive` flag to any command to make sure it will never prompt, or set a global `ZOS_NON_INTERACTIVE` environment variable. The CLI will also honor the `DEBIAN_FRONTEND=noninteractive` variable.
+
+## New unpack command
+
+A new command `zos unpack` was added for unpacking a zepkit instance. We are providing two packs initially: 
+- `zos unpack zepkit` will initialize a new barebones react app, preconfigured with OpenZeppelin, ZeppelinOS, Truffle, and Infura, ready to start coding.
+- `zos unpack tutorial` will initialize the same pack, but with an additional guided tutorial for you to get acquainted with it.
+
+Additionally, `zos unpack` can be directed to any github repository with a `kit.json` specification, so anyone in the community can propose their own packs to be used with ZeppelinOS.
```

Comment:
Is anyone writing a guide on how to make your ZeppelinOS kit? I'll do it if needed!

Reply:
It's on our tech writer's backlog. Most likely we'll ask for your help to review it.